src/eXpansion/Bundle/AdminChat/ChatCommand/ReasonUserCommand.php

```diff
@@ -113,8 +111,6 @@ public function __construct(
      */
     protected function configure()
     {
-        parent::configure();
```

Comment:
Why not ?

Reply:
it's not needed, there's nothing to override

Reply:
There is nothing to overide today but that method might have something to put back.